frontend/src/metabase/dashboard/components/DashCard/DashCard.tsx

```diff
@@ -362,10 +362,7 @@ function DashCardInner({
       };
     } else {
       return {
-        state: getInitialStateForCardDataSource(
-          series[0].card,
-          series[0].data?.cols ?? [],
-        ),
+        state: getInitialVisualizerStateForMultipleSeries(series),
```

Comment:
Does this mean it won't work for non multiple series cards anymore?

Reply:
I initially thought this function can be used even for a single series, however, now I see how much the logic has changed for it so I separated this case https://github.com/metabase/metabase/pull/56043/files#diff-d097690002e11cf2efe0b8be8908648521eeaf1976b5643c854906c626530feeR325